tasks/run-check-cem.js

```diff
@@ -78,23 +76,85 @@ const ignoredPackages = [
     '@types/swc',
 ];
 
-// Get the list of workspace packages
-const allPackages = getWorkspacePackages(ignoredPackages);
-
-// Execute check-cem.js for each package except the ignored ones
-const checkCemPath = path.resolve(__dirname, 'check-cem.js'); //'../../tasks/check-cem.js';
-allPackages.forEach((pkg) => {
-    if (fs.existsSync(checkCemPath)) {
-        console.log(`Running check-cem.js for ${pkg.name}`);
-        try {
-            execSync(`node ${checkCemPath}`, {
-                stdio: 'inherit',
-                cwd: pkg.path,
-            });
-        } catch (error) {
-            console.error(`Error running check-cem.js for ${pkg.name}:`, error);
-        }
-    } else {
-        console.warn(`check-cem.js not found for ${pkg.name}`);
+async function checkCEM(pkg) {
```

Comment:
I pulled this functionality directly into the run script here since it seemed like an unnecessary abstraction.